src/cli/commands/build/build.ts

```diff
@@ -1,15 +1,12 @@
-import path from "path";
+import path from "node:path";
 import chalk from "chalk";
-import { detectProjectFolders, generateConfiguration } from "../../../core/frameworks";
-import {
-  findUpPackageJsonDir,
-  isUserOrConfigOption,
-  logger,
-  pathExists,
-  readWorkflowFile,
-  runCommand,
-  swaCliConfigFilename,
-} from "../../../core/utils";
+import { detectProjectFolders, generateConfiguration } from "../../../core/frameworks/detect.js";
+import { findUpPackageJsonDir, pathExists } from "../../../core/utils/file.js";
```

Comment:
Would it be worth it to change the `src/utils/index.ts` file to include the exports of all of the utility functions? I know it's a lot of functions but we would be able to import all of the required functions more easily like in the original diff rather than splitting the imports across 6 files.

The same would go for the other sub-folders where `index.ts` was removed as well. Let me know your thoughts!

Reply:
Consensus from the TS team is that since you have to import ".../index.js" now, it's not as worth it as it was when JS had directory focused imports.  It would be worthwhile implementing root-relative and directory-index imports as a separate item, but that requires switching away from tsc.  Suggest we leave it alone for right now.